build-system/global-configs/experiments-config.json

```diff
@@ -3,6 +3,7 @@
     "name": "ANALYTICS_VENDOR_SPLIT",
     "command": "gulp dist --defineExperimentConstant=ANALYTICS_VENDOR_SPLIT",
     "issue": "",
+    "expirationTimestampMs": "1565870400000",
```

Comment:
hmmm the value is not very readable. Is it possible to have the value follow some sort of readable format?

Reply:
we can also do `2019-01-15`, which follows year-month-day format

Reply:
Sure I like the idea. May need to also agree on timezone as well. I wonder if there's a UTC date format that we can follow. Could you please take a look? Thanks.

Reply:
Using this date-only format in the `Date` constructor [defaults to UTC timezone](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse#Date_Time_String_Format) :)